cli/internal/util/env/env_test.go

```diff
@@ -0,0 +1,147 @@
+package env
+
+import (
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestGetHashableEnvPairs_SingleVar(t *testing.T) {
+	t.Setenv("lowercase", "stillcool")
+	t.Setenv("MY_TEST_VAR", "cool")
+	t.Setenv("12345", "numbers")
+	t.Setenv("one_MORE", "great")
+	println("cool")
```

Comment:
```suggestion
```

Reply:
Haha you must have been fast on this review, this didn't last in here for long.